db/schema.rb

```diff
@@ -42,6 +42,9 @@
     t.string "name"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.text "description"
+    t.string "kindofsport"
+    t.boolean "private"
```

Comment:
See above: If you make the `private` field mandatory, it should have a `null: false` attribute in the db schema (as well as in the migration).